tcpClientServer/tests/tst_tcpclientserver.cpp

```diff
@@ -40,6 +42,27 @@ void tcpClientServer::clientFailedInitTest() {
     clientProcess.waitForFinished();
     QCOMPARE(clientProcess.exitCode(), QProcess::CrashExit);
 }
+
+void tcpClientServer::succeededInitTest() {
+    QProcess serverProcess;
+    serverProcess.start(QString("../tcp-server"), QStringList());
+    serverProcess.waitForStarted();
+
+    QTest::qSleep(1000);
```

Comment:
Ti è capitato di veder fallire il test senza questa?
1 secondo di attesa per un singolo test non è poco: se avessimo un secondo di sleep in ciascun test per ciascun test di unità, finiremmo per aspettare un sacco di tempo per ottenere i risultati nel caso di un progetto mediamente grande.
Se la sleep serve allora ok, magari cerchiamo di dimensionarla per ridurre al minimo le attese inutili, se invece non serve non mettiamola.